### ISSUE TITLE
frame_stack default to False

### ISSUE THREAD

**Issue by lweitkamp** [higgsfield/RL-Adventure#9]
Hello, is it correct that frame_stack in wrap_deepmind is never in use? Will you be able to get velocity information if you only parse 1 frame at a time?

**Comment by garkavem** [comment]
Yeah, it seems like it is never in use. I wonder, how the agent manages to learn to play pong.
